blog: add fluentd v1.16.8 release announcement (#371)
fluent/fluentd-website: content/blog/20250501_fluentd-v1.16.8-has-been-released.md

# Fluentd v1.16.8 has been released

Hi users!

We have released v1.16.8 on 2025-05-01.
ChangeLog is [here](https://github.com/fluent/fluentd/blob/v1.16/CHANGELOG.md#release-v1168---20250501).

This release is a maintenance release of v1.16 series.

This release will be bundled for upcoming `fluent-package` LTS version v5.0.7!

## Bug Fixes

> ### Windows: Stop the service when the supervisor is dead

Reviewer: ```suggestion
### Windows: Stop the service correctly when the supervisor is dead
```

In this release, we have fixed a bug that Windows service process will continue running
even if the supervisor process is dead due to config error or etc.

> It is easier to notice config errors and other issues by this issue fixed.

Reviewer 2: ```suggestion
With this issue fixed, it becomes easier to notice config errors and other issues.
```

This seems easier to understand!

### `formatter_csv`: Fixed memory leak

> The caching mechanism introduced for performance optimization that was caching threads inappropriately in `formatter_csv`.

Reviewer 2: > The caching mechanism introduced for performance optimization that was caching threads inappropriately in `formatter_csv`.

I find it difficult to read this sentence.
Which is the verb?

Author: Subject is `The caching mechanism introduced for performance optimization`.

Author: verb is `was caching`

Author: Removed

The `in_exec` plugin creates a thread each time to emit an event.
Then all threads were cached in `formatter_csv`, causing a memory leak.

> With this issue fixed, it becomes stable.

Reviewer: How about adding  reference link to https://github.com/fluent/fluentd/pull/4920

Reviewer 2: If adding a ref to this, then shouldn't we add a ref to https://github.com/fluent/fluentd-website/pull/371/files#diff-3b0383239d4da17f3094b906614f349915ac1e9f94cf44e53c6038721281296cR19 as well?

Author: make sense.

Author: Added
